packages/mobile/ios/Quiet/AppDelegate.m

```diff
@@ -176,7 +176,7 @@ - (void)applicationDidEnterBackground:(UIApplication *)application
 {
   [self stopTor];
 
-  NSString * message = [NSString stringWithFormat:@""];
+  NSString * message = [NSString stringWithFormat:@"app:close"];
```

Comment:
This was just to get rid of an error that didn't break anything, it was just annoying.